src/Tasks/Copy.cs

```diff
@@ -274,28 +276,46 @@ private void LogDiagnostic(string message, params object[] messageArgs)
                 destinationFileExists = destinationFileState.FileExists;
             }
 
-            bool linkCreated = false;
+            bool symboliclinkCreated = false;
+            bool hardlinkCreated = false;
```

Comment:
nit:
symbolicLinkCreated and hardLinkCreated.

Reply:
Updated